### ISSUE TITLE
Cannot reprocess batch

### ISSUE THREAD

**Issue by ulsa** [awslabs/aws-lambda-redshift-loader#14]
I have a batch that is in status `locked`:

```
$ AWS_PROFILE=myprofile node queryBatches.js eu-west-1 locked
[
  {
    "s3Prefix": "mybucket/logs/access/cdn",
    "batchId": "ea3cbcdb-4526-4ba2-aaa4-9f12503f7b54",
    "lastUpdateDate": "2015-04-15-13:02:24"
  }
]
```

I run the `reprocessBatch.js` script:

``` bash
$ AWS_PROFILE=myprofile node reprocessBatch.js eu-west-1 ea3cbcdb-4526-4ba2-aaa4-9f12503f7b54 mybucket/logs/access/cdn
Submitted reprocess request for mybucket/logs/access/cdn/cdn_20150214_0001.log.gz
Processed 1 Files
```

I'm not sure exactly what should happen. There are no entries in the table `LambdaRedshiftProcessedFiles`. There is still only one entry in the table `LambdaRedshiftBatches`. There are no new log streams or events in CloudWatch.

**Comment by IanMeyers** [comment]
Hello,

Interesting. What is your batch size set to? Can you unlock it with unlockBatch.js?

Thx,

Ian

**Comment by ulsa** [comment]
Batch size is 1, since I have quite large files (50-200 MB). I'll try the unlock script. Didn't notice that before. It wasn't mentioned in the readme.

**Comment by IanMeyers** [comment]
Yes, was omitted for some reason but in the readme now.

**Comment by ulsa** [comment]
```
$ AWS_PROFILE=myprofile node unlockBatch.js eu-west-1 78cf9b0b-6426-49a8-be87-1d83b04641c7 mybucket/logs/access/cdn
Batch 78cf9b0b-6426-49a8-be87-1d83b04641c7 Unlocked and ready for reprocessing
$ AWS_PROFILE=myprofile node reprocessBatch.js eu-west-1 78cf9b0b-6426-49a8-be87-1d83b04641c7 mybucket/logs/access/cdn
Cannot reprocess an Open Batch
```

**Comment by IanMeyers** [comment]
Sure - you can now just add more files to S3 and this will be added to the current open batch.

**Comment by ulsa** [comment]
Even if the batch size is 1?

**Comment by IanMeyers** [comment]
Certainly will. It will start processing in the next file.

**Comment by ulsa** [comment]
OK, so the batch is open. I tried uploading the same file again, but then I get "Already Processed". I run `processedFiles -d` on it and upload it again. Nothing happens.

I tried uploading another file, in case the open batch only were to be closed if the batch size was exceeded, but it just creates a new batch, that ends up in locked state due to the decrypting problem discussed in issue #16. 

So I have now one batch that's open and one that's locked. Not sure what I should have done differently.
